gh-skeleton

```diff
@@ -107,6 +107,14 @@ clone_repo() {
   log_info "Cloning skeleton remote repository to the new local repository."
   git clone --origin "${src_repo}" git@github.com:"${src_org}/${src_repo}".git "${dest_repo}"
 
+  log_info "Adding a new remote origin for the repository."
+  git -C "${dest_repo}" remote add origin git@github.com:"${dest_org}/${dest_repo}".git
+
+  log_info "Setting base repository for pull request and issue creation."
+  # TODO: Replace with a gh command when it is implemented.
```

Comment:
You should create an issue in this repo so that we don't lost track of this TODO item.  After the issue is created, please include the URL here in a comment.

Reply:
See: 
- https://github.com/cisagov/gh-skeleton/issues/17